addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
From a forward-maintainability standpoint, this override seems to omit certain features in newer versions of ember that might be relevant (and even helpful for your specific use case).

http://emberjs.com/blog/2016/06/09/ember-data-2-6-released.html#toc_code-ds-improved-ajax-code-a-href-https-github-com-emberjs-data-pull-3099-3099-a

https://github.com/emberjs/data/blob/master/addon/adapters/rest.js#L630